daemon/xmusic-cli.py

```diff
@@ -22,8 +23,16 @@
 
     with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
         sock.connect(server_addr)
-        sock.sendall(bytes(request + "\n", "utf-8"))
-        response = str(sock.recv(1024), "utf-8")
+        header = pack("=BHB", 29, len(request), 29)
+        sock.sendall(header + bytes(request, "UTF-8"))
+
+        length = unpack("=BHB", sock.recv(4))[1]
+        response = ""
+        while len(response) < length:
+            buf = sock.recv(min(1024, length - len(response)))
```

Comment:
The value of `1024` may use a constant to unified management. ex. `RPCHandler. MAX_BUF_LENGTH`.